src/cr/cube/cubepart.py

```diff
@@ -614,6 +614,15 @@ def pairwise_indices(self):
         column in the same row with a confidence interval meeting the threshold defined
         for this analysis.
         """
+        # If overlaps are defined, calculate significance based on them
+        if (
+            self.dimension_types[-1] == DT.MR
+            and self._cube.overlaps is not None
+            and self._cube.valid_overlaps is not None
+        ):
+            return self._assembler.pairwise_indices(self._alpha, self._only_larger)
+
+        # If overlaps are not defined, default to the legacy way of calculating sig
```

Comment:
I think that this change is needed also in the `pairwise_indices_alt` to enable the 2 level (aBCdeF) pairwise sig letters

Reply:
done